fontir/src/ir.rs

```diff
@@ -1,18 +1,27 @@
-//! Serde types for font IR. See TODO:PublicLink.
+//! Serde types for font IR.
 
 use crate::error::Error;
 use ordered_float::OrderedFloat;
 use serde::{Deserialize, Serialize};
 use std::collections::{BTreeMap, HashMap};
 
-///Global font info that cannot vary.
+/// Global font info that cannot vary.
 ///
 /// For example, upem, axis definitions, etc, as distinct from
 /// metadata that varies across design space such as ascender/descender.
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
 pub struct StaticMetadata {
-    axes: Vec<Axis>,
-    glyph_order: Vec<String>,
+    pub axes: Vec<Axis>,
+    pub glyph_order: Vec<String>,
+}
+
+impl StaticMetadata {
+    pub fn glyph_id(&self, name: &str) -> Option<u32> {
+        self.glyph_order
+            .iter()
+            .position(|gn| gn == name)
+            .map(|pos| pos as u32)
```

Comment:
wonder if we could/should cache this into a HashMap<String, u32> so glyph_id lookup is O(1) instead of O(n)

Reply:
We should, I just wanted to start with naively correct.

Reply:
Added